tests/PasswordGeneratorTest.php

```
@@ -0,0 +1,215 @@
+            "Should include at least '1' digit character.",
+        );
+        self::assertSame(
+            1,
+            preg_match('/^[a-zA-Z0-9!@#$%^&*()_\-=+;:,.?]+$/', $password),
+            'Should contain only characters from the configured password pool.',
+        );
+    }
+
+    public function testGenerateUsesArraySpliceWithSingleItemRemoval(): void
+    {
+        PasswordGenerator::generate(4);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\PHPForge\Helper\PasswordGenerator' in method 'testGenerateUsesArraySpliceWithSingleItemRemoval'. (undefined)

(StaticAccess)